packages/strapi-provider-email-nodemailer/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "strapi-provider-email-nodemailer",
+  "version": "3.0.0",
```

Comment:
`v3.2.5`

Reply:
We have a script to handle the versioning, but yeah probably not a bad idea to set it to the current version so the script picks it up

Reply:
Done. Is it common to use the same version for all packages in a mono-repo? What would have happened if the library had a version `> 3.2.5` already?